internal/providers/github/properties/artifact.go

```diff
@@ -108,21 +108,25 @@ func getNameFromParams(owner, name string) string {
 	return prefix + name
 }
 
-func parseArtifactName(name string) (string, string, string, error) {
+func parseArtifactName(name string) (owner string, artifactName string, artifactType string, err error) {
 	index := strings.Index(name, "/")
 	if index == -1 {
 		// No slash found, treat the entire name as the artifact name
-		return "", name, string(verifyif.ArtifactTypeContainer), nil
+		artifactName = name
+		artifactType = string(verifyif.ArtifactTypeContainer)
+		return
```

Comment:
I'm fine with this style, as I think named return parameters can have good use cases, but I feel like someone else didn't like them.  Maybe @JAORMX ?

Reply:
Not me in this case, it was my suggestion in the aforementioned previous review.